src/panels/config/voice-assistants/ha-config-voice-assistants-assistants.ts

```diff
@@ -36,6 +37,8 @@ export class HaConfigVoiceAssistantsAssistants extends LitElement {
         .tabs=${voiceAssistantTabs}
       >
         <div class="content">
+          <assist-pref .hass=${this.hass} dir=${computeRTLDirection(this.hass)}>
```

Comment:
does it need the `dir`?

Reply:
Not anymore because I removed the switch